README

```diff
@@ -1,6 +1,23 @@
+# pam-config
+
 pam-config is a command line utility to maintain the common PAM
 configuration files included by most PAM application configuration
 files. It can be used to configure a system for different network or
 hardware based authentication schemes. pam-config can also
 add/adjust/remove other PAM modules and their options.
 
+## Contributing
+
+```bash
+sudo zypper install autoconf make gcc autogen gettext-tools
+```
+
+```bash
+cd pam-config
+autoreconf -fiv
+automake --add-missing
```

Comment:
Instead of autoreconf and automake you should call `./autogen.sh`
Or does this not work for you?

Reply:
Ahhh, I'm used to other projects where that isn't present. I'll update this.